index.html

```diff
@@ -80,5 +80,192 @@ <h2 id="h2"><p>I'm a Software Developer.</p></h2>
 
     <base1 id="base1"></base1>
 
+
+    <recentwork id="recentwork"><p id="recentwork_text">My Recent Works</p>
+    <img src="./assets/Vector.svg" alt="">
```

Comment:
Kindly, I suggest that for accessibility every time you use an "IMG" you indicate an appropriate "ALT" to describe the content of that image

Reply:
Done.